wearipedia/devices/fitbit/fitbit_sense.py

```diff
@@ -89,21 +90,75 @@ def _filter_synthetic(self, data, data_type, params):
         num_days_start = delta1.days
         num_days_end = delta2.days
 
-        return data[num_days_start : -num_days_end + 1]
+        if data_type == "sleep":
```

Comment:
All of the following if statements can be collapsed as intermediary = data[0][data_type][num_days_start : -num_days_end+1], of course with additional if statement to check if data_type is in the dictionary in the first place. THen just return [{data_type: intermediary}]